sdk/marketplaceordering/Microsoft.Azure.Management.MarketplaceOrdering/src/Generated/MarketplaceAgreementsOperations.cs

```diff
@@ -193,14 +193,13 @@ public async Task<AzureOperationResponse<AgreementTerms>> GetWithHttpMessagesAsy
             string _responseContent = null;
             if ((int)_statusCode != 200)
             {
-                var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
```

Comment:
This will result in a breaking change, please provide the corresponding spec PR.  The spec PR either needs the approved-breakingchange label from the breaking change team OR we will need to find another way to do this.  One option is if ErrorResponseException inherits from CloudException

Reply:
the PR link: https://github.com/Azure/azure-rest-api-specs/pull/12469
This is indeed a PR with breaking changes, a new version of the swagger was introduced in the PR because of this.
Can you advise on how to get an approval for this from the breaking change team?

Reply:
On the spec PR it has a link which should get you to the right place.  I have added it here for your convenience http://aka.ms/bcforapi